src/EditorFeatures/Core/project.json

```diff
@@ -1,6 +1,10 @@
 ﻿{
   "dependencies": {
-    "System.Collections.Immutable": "1.1.37"
+    "System.Collections.Immutable": "1.1.37",
```

Comment:
Why is this referencing Immutable? Shouldn't it flow from downlevel project refs?